tasks/cephfs/test_volume_client.py

```diff
@@ -64,6 +64,31 @@ def _configure_vc_auth(self, mount, id_name):
         self._sudo_write_file(mount.client_remote, mount.get_keyring_path(), out)
         self.set_conf("client.{name}".format(name=id_name), "keyring", mount.get_keyring_path())
 
+    def test_default_prefix(self):
+        group_id = "grpid"
+        volume_id = "volid"
+        guest_entity = "guest"
+        DEFAULT_VOL_PREFIX = "/volumes"
+        DEFAULT_NS_PREFIX = "fsvolumens_"
+
```

Comment:
First, you'd need to set up auth credentials for the client 'mount_b' such that it's a handle for driving the volumeclient.
Please add the following lines, 
self.mount_b.umount_wait()                                                                       self._configure_vc_auth(self.mount_b, "manila")